docker/conf/docker-compose-prod-deploy.yml.dist

```diff
@@ -0,0 +1,63 @@
+version: "3.1"
+services:
+    webserver:
+        restart: always
+        image: nginx:1.13-alpine
+        container_name: production-webserver
+        volumes:
+            - /var/www/production-content:/var/www/html/web/content
```

Comment:
What is production-content? Why you use fixed path for mounting it, how it gots created, where is written anything about it?

Reply:
all about this should be on line 238
https://github.com/shopsys/production/pull/1/files#diff-16a639ab3ed13ad89bdb5d906e50fbcfR238